dotnet/src/Connectors/Connectors.OpenAI/OpenAIServiceCollectionExtensions.cs

```diff
@@ -65,61 +78,68 @@ public static class OpenAIServiceCollectionExtensions
     /// </summary>
     /// <param name="services">The <see cref="IServiceCollection"/> instance to augment.</param>
     /// <param name="deploymentName">Azure OpenAI deployment name, see https://learn.microsoft.com/azure/cognitive-services/openai/how-to/create-resource</param>
-    /// <param name="modelId">Model identifier, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
     /// <param name="endpoint">Azure OpenAI deployment URL, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
     /// <param name="apiKey">Azure OpenAI API key, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
     /// <param name="serviceId">A local identifier for the given AI service</param>
     /// <returns>The same instance as <paramref name="services"/>.</returns>
     public static IServiceCollection AddAzureOpenAITextGeneration(
         this IServiceCollection services,
         string deploymentName,
-        string modelId,
         string endpoint,
         string apiKey,
+        string? serviceId = null) => AddAzureOpenAIChatCompletion(services, new() { DeploymentName = deploymentName, Endpoint = endpoint, ApiKey = apiKey }, serviceId);
```

Comment:
Looks like this should be `AddAzureOpenAITextGeneration`:
```suggestion
        string? serviceId = null) => AddAzureOpenAITextGeneration(services, new() { DeploymentName = deploymentName, Endpoint = endpoint, ApiKey = apiKey }, serviceId);
```